### PR TITLE
LPS-168109 Announce full path instead of item's label/type

### PR DESCRIPTION
**Pull request by sandrodw3**
This is a follow-up of https://github.com/liferay-echo/liferay-portal/pull/10298 that introduces an improvement suggested by @ethib137. From now we will announce the ful item path instead of item's label/type.

### CODE REVIEW

**Comment by ethib137** [comment]
Hey @sandrodw3, is the full item path translated?

**Comment by sandrodw3** [comment]
> Hey @sandrodw3, is the full item path translated?

That's a good point @ethib137! I updated the PR translating the item types 😄

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/components/KeyboardDNDContext.tsx [hunk 1]
```diff
@@ -629,11 +629,49 @@ function getTextData(
...
+			itemType === 'page' &&
+			paginationMode === 'single-page' &&
+			itemPath.length > 1
+		) {
+			return;
+		}
+
+		const itemName = item.label || `${itemTypeLabel} ${itemIndex + 1}`;
+
+		path = path ? [path, itemName].join(',') : itemName;
```
Comment (p2kmgcl) [inline modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/components/KeyboardDNDContext.tsx:652]: We could build this using a map and then joining every thing with a single `path.join(', ')` :smile:

Comment (sandrodw3) [reply modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/components/KeyboardDNDContext.tsx]: As discussed in slack, I finally refactored a bit and changed it to `push` + `join` 😄

**Comment by sandrodw3** [comment]
ci:forward

**Pull request closed without merging**